### ISSUE TITLE
GUI 2.2.0 vs GUI 1.5.1 - issues creating custom classifier

### ISSUE THREAD

**Issue by Bry0ny** [birdnet-team/BirdNET-Analyzer#791]
I have been building custom classifiers with the GUI version 1.5.1 and have had great results. I recently tried to use the exact same training/ testing data in the new GUI (version 2.2.0), but I'm getting conflicting results. In fact, the new GUI version is essentially detecting almost all of my 1 hr test data as the target species. 

- I have made sure to use the exact same parameters when training and have checked the  "CustomClassifer_Params" file that is created alongside the classifier, and this reports exactly the same parameters used. 
- The issue seems to lie with how the recogniser is being created, because I used my old classifier (created in GUI V1.5.1) in a batch analysis in the new GUI, and this worked fine. 
- I have made sure to use the same thresholds and other parameters in testing 
- I have tried re-downloading the new GUI version 

Another thing to note is that when I click "Analyze" in the batch analysis tab, I get the message "invalid audio files" rather than the usual successful run notification. It does give me the outputs still in the output directory. I've noticed the number of detections in each selection table changes between runs. 
- I have tried re-downloading the audio files to ensure they are not corrupted 

Perhaps I have missed something? Apologies if this has been asked already. 
Any ideas would be very much appreciated! 

Thank you kindly, 
Bryony

**Comment by Josef-Haupt** [comment]
Which training settings did you use? There was a breaking change in version 2.0, which means some settings might now lead to different results.

**Comment by Bry0ny** [comment]
Hi [Josef-Haupt](https://github.com/Josef-Haupt), thank you for looking  into this. 

My training parameters are below, they are essentially the default settings. I've checked these against the training parameters reported for V1.5.1 and they are both reported the same. 

You'll notice I've set the learning rate to 0.001 instead of the default 0.0001. I tried both in case this was what was causing the issue but it doesn't appear so. 

Let me know if you require any additional info.
Many thanks, 
Bryony 

<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:x="urn:schemas-microsoft-com:office:excel"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Excel.Sheet>
<meta name=Generator content="Microsoft Excel 15">
<link id=Main-File rel=Main-File
href="file:////Users/bryonymargetts/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip.htm">
<link rel=File-List
href="file:////Users/bryonymargetts/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">
<style>

</style>
</head>

<body link="#467886" vlink="#96607D">


Hidden units | Dropout | Batchsize | Learning rate | Crop mode | Crop overlap | Audio speed | Upsamling mode | Upsamling ratio | use mixup | use label smoothing | BirdNET Model version
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
0 | 0 | 32 | 0.001 | center | 0 | 1 | repeat | 0 | FALSE | FALSE | V2.4



</body>

</html>

**Comment by Bry0ny** [comment]
Hi @kahst 
Thank you for your help - it's greatly appreciated. 
I have sent you an email containing my training/testing data that I have been using to look into this difference in GUI versions. Perhaps I am making some error on my end, my apologies if so! 
 
I will try your other recommendations tomorrow when I am back in the office.

**Comment by Bry0ny** [comment]
Hi, no worries at all, I've sent the ground-truth labels via email (the same address as the training files). 
I did notice the training stopping relatively early, yes - perhaps something related to that. 

Thanks for clarifying the "invalid audio files". After using the old GUI for so long the change was leading me to believe there was an error here, but that makes sense. Good to know, thank you! 

Thanks once again for looking into this for me - Cheers, Bryony
